std/net/curl.d

```diff
@@ -1069,12 +1121,23 @@ private auto _basicHTTP(T)(const(char)[] url, const(void)[] sendData, HTTP clien
             import std.conv : to;
             content.reserve(value.to!size_t);
         }
+        header ~= cast(ubyte[])(key ~ ": " ~ value ~ "\n");
```

Comment:
Doesn't this add a lot of overhead if you aren't interested in head? Allocations aren't for free